lib/bridge/pbn/version.rb

```diff
@@ -0,0 +1,2 @@
+require_relative '../../bridge.rb'
+module Bridge::Pbn; VERSION = "0.1.0"; end
```

Comment:
You could simply skip the `Bridge` namespace all together, since this is just a `Pbn` library.